test/git_repository.go

```diff
@@ -72,6 +72,15 @@ func NewGitRepository(workingDir string, homeDir string, shell command.Shell) Gi
 	return GitRepository{Dir: workingDir, Shell: shell}
 }
 
+// AnyBranchesOutOfSync returns if true if and only if one or more local branches are out in sync with their remote
+func (repo *GitRepository) AnyBranchesOutOfSync() (bool, error) {
```

Comment:
What do you think about the convention to name functions that return a bool either `Is...` or `Has...`? For example, `IsValidUser`, or `HasBranchesOutOfSync`? Having consistency here helps locate the right methods and avoid duplicate implementations.

Reply:
Updated